app/views/admin/invoices/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>All Invoices</h1>
+
+<% @invoices.each do |invoice| %>
+  <h3>Invoice #<%= link_to invoice.id, admin_invoice_path(invoice) %></h3>
+<% end %>
```

Comment:
We should make sure we figure out a way to handle the header/nav bar in the app layout. I've been writing them into to the views but I can go back and fix them once we have that figured out.

Reply:
sounds good. please teach me what to do!!